src/main/java/umaikaze/duke/Storage.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Storage class saves List<Task> in the directory providing upon initialization
+ * Loads from file, parse it using Parser and returns a List<Task>
+ */
+
+package umaikaze.duke;
+
+import umaikaze.duke.task.Deadline;
+import umaikaze.duke.task.Event;
+import umaikaze.duke.task.Task;
+import umaikaze.duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private File saveFile;
+
+    /**
+     * @param savePath This is the only way to set save directory for each Storage object
+     */
+    public Storage(String savePath) {
+        this.saveFile = new File(savePath);
+    }
+
+    /**
+     * Adds Tasks from list to the data file as specified by saveFile
+     * WARNING: In the case where the file is already populated, this method would not delete the original content
+     * but simply append new Tasks to the file
+     */
```

Comment:
Since this method takes in parameters and throws exceptions, it would be better to use javadoc's "@param" and "@exception" tags so you have more accurate javadoc generation